pick.lic

```diff
@@ -112,8 +112,9 @@ class LockPicker
   end
 
   def buy_lockpick(type)
-    fput("buy #{type} lockpick")
-    fput("offer #{@lockpick_costs[type]}")
+    response = dothis("buy #{type} lockpick", /prepared to/)
```

Comment:
I have a preference for bput since it's DR specific errors, has a timeout (although there's a dothistimeout), and only returns the matched substring sent, not the entire message. Switch to bput here would require expanding the match message to the regex below, however that does make it simpler to reuse the regex in both places.